app/src/main/java/org/wikipedia/page/tabs/SwipeableTabTouchHelperCallback.kt

```diff
@@ -0,0 +1,65 @@
+package org.wikipedia.page.tabs
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import androidx.recyclerview.widget.ItemTouchHelper
+import androidx.recyclerview.widget.RecyclerView
+import org.wikipedia.R
+import org.wikipedia.util.ResourceUtil.getThemedColor
+import kotlin.math.abs
+
+class SwipeableTabTouchHelperCallback(context: Context) : ItemTouchHelper.Callback() {
+
+    interface Callback {
+        fun onSwipe()
+        fun isSwipeable(): Boolean
+    }
+
+    private val swipeBackgroundPaint = Paint()
+    private val itemBackgroundPaint = Paint()
+    var swipeableEnabled = false
+
+    init {
+        swipeBackgroundPaint.style = Paint.Style.FILL
+        swipeBackgroundPaint.color = getThemedColor(context, R.attr.background_color)
```

Comment:
Can we use `ResourceUtil.getThemedColor` here?

Reply:
yep!